app/config/findbugs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
+<!--
+FindBugs exclude filter
+Severity: HARD
+-->
+<FindBugsFilter>
```

Comment:
He has just added those out of paranoia rather than facts. As you can see already, the build hasn't failed. Meaning that find bugs is not parsing those files at all